### DISCUSSION TITLE
The editor cache is gone...

### DISCUSSION THREAD

**Discussion by TinyRei** [usememos/memos#2205] [Q&A] [answered]
@boojack 

大佬，你好，从v0.13.2版本开始，编辑框的本地缓存功能消失了，这个功能特别有用，可以让它回来嘛？

Hi.
There is no editor local cache from the version 0.13.2.
We need local storage on memo editor, or it's better for us to control it open.
Pls

**Comment by 8mnl** [comment, marked as answer]
Editor cache was added in [0.15.0](https://github.com/usememos/memos/releases/tag/v0.15.0) now @TinyRei :)
